include/wx/msw/toplevel.h

```diff
@@ -191,6 +194,9 @@ class WXDLLIMPEXP_CORE wxTopLevelWindowMSW : public wxTopLevelWindowBase
     wxWindowRef m_winLastFocused;
 
 private:
+    // dpi related
```

Comment:
I think a slightly more detailed comment here, notably explaining whether `m_activeDPI` is used if we're not per monitor DPI aware and when is it changed, would be useful.